examples/create-java-objects/create-java-objects.description

```diff
@@ -0,0 +1,8 @@
+// jBallerina is an implementation of the Ballerina language specification that targets the JVM.
```

Comment:
The BBE generation tool looks for a .description file with the same name as the directory, but with the "-"s replaced by "_"s. 

`create_java_objects.description`

Shall we also use underscores for the .bal and .out files?

Reply:
Oh, I missed that :(  Let me update the PR now. Thanks

Reply:
Done. Thanks.